instaclustr/resource_bundle_user.go

```diff
@@ -0,0 +1,166 @@
+package instaclustr
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceBundleUser() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBundleUserCreate,
+		Read:   resourceBundleUserRead,
+		Update: resourceBundleUserUpdate,
+		Delete: resourceBundleUserDelete,
+
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
```

Comment:
All the properties that we don't expect to be updated should be marked as ForceNew. In this case - cluster_id, bundle_name should be marked.

Reply:
done